Assignment4.txt

```diff
@@ -0,0 +1,13 @@
+//Anya Lauria
+
+1. A character array is a data structure of which the elements are of data
```

Comment:
a string is NOT a data type. A string is a char array, but with a null terminator as its last element.